include/api.php

```diff
@@ -2851,7 +2852,7 @@ function api_direct_messages_new($type) {
 	function api_direct_messages_destroy($type){
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.